### PR TITLE
ColorizingStreamHandlerMixin: Allow force/forbid colors

### PR DESCRIPTION
**Pull request by ayalash**

### CODE REVIEW

**Comment by ayalash** [comment]
@vmalloc Can you please review?

File: logbook/more.py [hunk 1]
```diff
@@ -329,6 +329,13 @@ class ColorizingStreamHandlerMixin(object):
 
     .. _`colorama`: https://pypi.python.org/pypi/colorama
     """
+    _use_color = None
+
+    def force_color(self):
```
Comment (vmalloc) [inline logbook/more.py]: Please add docstrings to the new methods

File: tests/test_more.py [hunk 2]
```diff
@@ -31,11 +31,16 @@ def test_colorizing_support(logger):
     from logbook.more import ColorizedStderrHandler
 
     class TestColorizingHandler(ColorizedStderrHandler):
+        def __init__(self, *args, **kwargs):
+            self._obj_stream = StringIO()
+            super(TestColorizingHandler, self).__init__(*args, **kwargs)
```
Comment (vmalloc) [inline tests/test_more.py]: Please make the call to `super` first

**Comment by ayalash** [comment]
@vmalloc I fixed your comments. Please review.

**Pull request merged**